### ISSUE TITLE
App rejected due to use of SDKs prohibited for children

### ISSUE THREAD

**Issue by pavelkukov** [expo/expo#7885]
I'm building a coloring book app for children. However, the app is rejected (multiple times) due to the usage of SDKs prohibited for children. This is for Amazon appstore.

The exact text is:
> Your app submission does not meet one or more of our acceptance criteria for some or all targeted devices. Failure reason(s) are listed below:
Your app appears to be directed at children, and contains an SDK(s) whose terms of use prohibit its use in children’s apps. Please remove the SDK AMPLITUDE and resubmit the app. Alternatively, If you use a restricted feature set of this SDK, please send a confirmation with the details (such as details about your data collection practices or a screenshot of the restricted feature set toggle in your SDK configuration) by visiting the Contact Us page and selecting the category “Appstore” and the Topic “Content Policy Results”.
You can view our guidelines on the Appstore Developer Portal.

Here is a link to the guidelines: https://developer.amazon.com/docs/policy-center/privacy-children.html#sdks-in-apps-for-children

The problematic AMPLITUDE SDK is indeed part of the app. See: https://github.com/expo/expo/blob/master/ios/Exponent/Supporting/kernel-manifest.json
```
expo-analytics-amplitude
"extra":{"amplitudeApiKey":"081e5ec53f869b440b225d5e40ec73f9"}
```

Is there a way to remove it?

**Comment by cruzach** [comment]
Hi @pavelkukov, you can't remove the Amplitude SDK from a managed workflow project. Although, I should mention that we've had this report before by other teams, and after they contacted the development support team from Amazon, and explained that although the Amplitude SDK is bundled in by Expo, they are not actually using it in their app at all, they were approved. You may want to try giving this a shot

Otherwise, to control exactly what native code is and isn't in your project, you should use the [bare workflow](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/#bare-workflow)

Good luck in getting Amazon app store approval!

**Comment by pavelkukov** [comment]
OK, thanks, @cruzach ! 
I've tried to explain to the amazon support, but without a result, for now. Still awaiting the response.

Form the docs
> In the bare workflow the developer has complete control, along with the complexity that comes with that. ... build service, notifications, over-the-air updates, and easy configuration with app.json are not yet supported

I'm don't want to eject to the bare workflow. :)

Thanks again for clarifying!

**Comment by cruzach** [comment]
Notifications and Updates are now available in the bare workflow - https://blog.expo.io/expo-sdk-37-is-now-available-dd5770f066a6

I'll update the docs :)
